cmd/krew/cmd/update.go

```diff
@@ -41,12 +57,147 @@ Remarks:
 	RunE: ensureIndexUpdated,
 }
 
+func retrieveUpdatedPluginList() ([]string, error) {
+	modifiedFiles, err := gitutil.ListModifiedFiles(constants.IndexURI, paths.IndexPluginsPath())
+	if err != nil {
+		return []string{}, err
```

Comment:
Please wrap the error with some meaningful message `errors.Wrapf(err, ...)`. Same in other places.